libr/include/r_cons.h

```diff
@@ -857,6 +864,7 @@ typedef struct r_selection_widget_t {
 	int w, h;
 	int scroll;
 	bool complete_common;
+	int direction;
```

Comment:
if we dont plan to support more than 2 directions maybe worth to use bool and remove the "enum"

Reply:
I just thought, it would increase the "readability" of the code. I will change it to bool :)